### DISCUSSION TITLE
Bug: DayPickerContextValue invalid onSelect field

### DISCUSSION THREAD

**Discussion by HEHOHADA** [gpbl/react-day-picker#1509] [Support] [answered]
### Bug description

Cannot get `onSelect`  that is described in the type DayPickerContextValue

<img width="795" alt="image" src="https://user-images.githubusercontent.com/42976899/184926252-ec2d7f92-4c68-42fa-b835-3d903be8daac.png">

- Add the ability to get a value
- remove it from the type

**Comment by gpbl** [comment]
@HEHOHADA some more details?

**Reply by HEHOHADA** [reply]
@gpbl  

The field 'onSelect' is described in the context value type, but it is not passed into the value

src/contexts/DayPicker/DayPickerContext.tsx

<img width="615" alt="image" src="https://user-images.githubusercontent.com/42976899/185173254-e7ef496f-fb04-4273-b5cf-35cd3cfc8cbe.png">

**Reply by gpbl** [reply]
@HEHOHADA thanks I see it now! Will work on a fix!

**Comment by gpbl** [comment, marked as answer]
This should be fixed in v8.1.1
